go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/pulumi/pulumi-yaml
 
-go 1.23
```

Comment:
Made this change because otherwise `go mod tidy` didn't work unless this was set